docs/reference/reference-interface/job-config-reference.rst

```diff
@@ -79,6 +79,25 @@ The following configuration settings are supported:
    -  ``drop_capabilities``: Just like ``add_capabilities`` but corresponding to the ``--cap-drop``
       argument of ``docker run`` rather than ``--cap-add``.
 
+   -  ``docker_flags``: A list of flags and arguments that ``docker run`` accepts. This is honored
+      by resource managers of type ``agent`` but ignored by others. The following arguments that
+      ``docker run`` accepts are not supported and will be ignored due to Determined specifying them
+      or providing a different method of setting them.
+
+      -  ``--user``
+      -  ``--expose``
+      -  ``--env``
+      -  ``--env-file``
+      -  ``--entrypoint``
+      -  ``--workdir``
+      -  ``--privileged``
+      -  ``--network``
+      -  ``--volume``
+      -  ``--shm-size``
+      -  ``--cap-add``
+      -  ``--cap-drop``
+      -  ``--device``
+
```

Comment:
-  ``docker_flags``: A list of flags and arguments accepted by ``docker run``. This is honored
      by resource managers of type ``agent`` but ignored by others. The following arguments, although accepted by
      ``docker run``, are not supported by Determined. Determined ignores these arguments because it either already specifies them or provides alternate methods for setting them.

      -  ``--user``
      -  ``--expose``
      -  ``--env``
      -  ``--env-file``
      -  ``--entrypoint``
      -  ``--workdir``
      -  ``--privileged``
      -  ``--network``
      -  ``--volume``
      -  ``--shm-size``
      -  ``--cap-add``
      -  ``--cap-drop``
      -  ``--device``